app/mva/forks.go

```diff
@@ -0,0 +1,125 @@
+package mva
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkauthkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	sdktypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting/exported"
+
+	authtypes "github.com/certikfoundation/shentu/v2/x/auth/types"
+	bankkeeper "github.com/certikfoundation/shentu/v2/x/bank/keeper"
+	stakingkeeper "github.com/certikfoundation/shentu/v2/x/staking/keeper"
+)
+
+func RunForkLogic(ctx sdk.Context, ak *sdkauthkeeper.AccountKeeper, bk bankkeeper.Keeper, sk *stakingkeeper.Keeper) {
+	ctx.Logger().Info("Applying Shentu MVA upgrade." +
+		" Fixing Shentu MVA accounts to correctly update to the right delegation tracking.")
+	e := FixAccounts(ctx, ak, bk, sk)
+	if e != nil {
+		panic(e)
+	}
+}
+
+func FixAccounts(ctx sdk.Context, ak *sdkauthkeeper.AccountKeeper, bk bankkeeper.Keeper, sk *stakingkeeper.Keeper) error {
+	var iterErr error
+
+	ak.IterateAccounts(ctx, func(account sdktypes.AccountI) (stop bool) {
+		mvacc, ok := account.(*authtypes.ManualVestingAccount)
+		if !ok {
+			return false
+		}
+
+		wb, err := MigrateAccount(ctx, mvacc, bk, sk)
+		if err != nil {
+			iterErr = err
```

Comment:
We can just panic here and remove the need to return an error?

Reply:
yeah we can